sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -423,6 +423,45 @@ AZ_NODISCARD az_result az_iot_provisioning_client_query_status_get_publish_topic
     size_t mqtt_topic_size,
     size_t* out_mqtt_topic_length);
 
+/**
+ * @brief Azure IoT Provisioning Client options for az_iot_provisioning_client_get_request_payload.
```

Comment:
Does this render correctly in doxygen and finds the link to the method?
OR
Do we need to do something like:
```suggestion
 * @brief Azure IoT Provisioning Client options for #az_iot_provisioning_client_get_request_payload.
```
Or even:
```suggestion
 * @brief Azure IoT Provisioning Client options for #az_iot_provisioning_client_get_request_payload().
```

Reply:
you can just have the `()`. Doxygen knows if it hits a `()` to find the documented function. No `#` needed.
```suggestion
 * @brief Azure IoT Provisioning Client options for az_iot_provisioning_client_get_request_payload().
```

Reply:
Fixing.  As I said in another comment I'll more closely review the doxygen output and fixup any rough spots I see.